### PR TITLE
Add KMS on CloudSQL module

### PR DESCRIPTION
**Pull request by lcaggio**

### CODE REVIEW

File: modules/cloudsql-instance/README.md [hunk 1]
```diff
@@ -93,6 +93,59 @@ module "db" {
...
+  name            = "my-db-project"
+  services = [
+    "servicenetworking.googleapis.com"
+  ]
+}
+
+resource "google_project_service_identity" "jit_si" {
+  provider   = google-beta
+  project    = module.project.project_id
+  service    = "sqladmin.googleapis.com"
```
Comment (juliocc) [inline modules/cloudsql-instance/README.md:113]: You're already adding `sqladmin` to  modules/project/service-accounts.tf. Just adding `sqladmin.googleapis.com` to the project's services should be enough to trigger this.

File: modules/cloudsql-instance/README.md [hunk 2]
```diff
@@ -93,6 +93,59 @@ module "db" {
...
+    name     = "keyring"
+    location = var.region
+  }
+  keys = {
+    key-sql = null
+  }
+  key_iam = {
+    key-sql = {
+      "roles/cloudkms.cryptoKeyEncrypterDecrypter" = [
+        "serviceAccount:${google_project_service_identity.jit_si.email}"
```
Comment (juliocc) [inline modules/cloudsql-instance/README.md:129]: Maybe we should add a `service_identities` output to `project`?

File: modules/cloudsql-instance/README.md (hunk 1 above)
Comment (juliocc) [reply modules/cloudsql-instance/README.md:113]: See #626

File: modules/cloudsql-instance/README.md (hunk 2 above)
Comment (juliocc) [reply modules/cloudsql-instance/README.md:129]: Ignore. See #626

File: modules/cloudsql-instance/README.md [hunk 3]
```diff
@@ -93,6 +93,59 @@ module "db" {
 }
 # tftest modules=1 resources=6
 ```
+
+### CMEK encryption pippo
```
[the remark this answers is not in the document]
Comment (juliocc) [reply modules/cloudsql-instance/README.md:97]: I understand now 😄. Just make sure to remove it before merging

[the remark this answers is not in the document]
Comment (lcaggio) [reply modules/cloudsql-instance/README.md:97]: LOL, it was just to easily trigger the test. I'll remove it

**Review by juliocc — APPROVED** [review]

**Pull request merged**